internal/resources/aws/codebuild_project.go

```diff
@@ -61,19 +60,28 @@ func (r *CodeBuildProject) nameLabel() string {
 	switch r.EnvironmentType {
 	case "WINDOWS_SERVER_2019_CONTAINER":
 		name = "Windows ("
-		name += strings.Replace(strings.ToLower(strings.SplitAfter(r.ComputeType, "BUILD_")[1]), "_", ".", 1) + ")"
+		name += r.computeType()
```

Comment:
This is going to return `Windows (` if `compute_type` is invalid. I'm not sure it's super important, because I'm guessing we don't find a price for it anyway, but it maybe we want something like:
```go
name = "Windows"
computeType := r.computeType()
if computeType {
  name += fmt.Sprintf(" (%s)", computeType)
}
```

Reply:
good catch, I'll update

Reply:
resolved in https://github.com/infracost/infracost/pull/1894/commits/5dce8342b27e25f8e31609ae0f8103f8715f7a16